main/systemd.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+    "fmt"
+    "os"
+    "os/exec"
+    "strings"
+    "path/filepath"
+    "github.com/pkg/errors"
+)
+
+func contains(distros []string, distro string) bool {
+    if distro != "" {
+        for _, v := range distros {
+            if v == distro {
+                return true
+            }
+        }
+    }
+    return false
+}
+
+func getDistro() (string, error) {
+    path, _ := os.Getwd()
+    output, err := exec.Command("sudo", filepath.Join(path, "scripts/distro_script.sh")).Output()
+    if err != nil {
+        return "", errors.Wrapf(err, "error running shell script")
+    }
+    distroName := strings.ToLower(string(output))
+    return distroName, nil
+}
+
+func GetSystemdUnitFileInstallPath() string {
+
+    var systemdPath string
+
+    distro, err := getDistro()
+
+    if (err != nil) {
+        errorMessage := fmt.Sprintf("Error getting distro name: %+v : %s", err, err.Error())
+	    errorLogger.Println(errorMessage)
```

Comment:
NIT: This looks like a formatting issue. Can you fix it?

Reply:
I noticed this. It's funny my IDE showing inline but GitHub shows formatting off.

Reply:
had to move this statement back and forth to figure out formatting issue